backend/routes_scanner.py

```
@@ -0,0 +1,137 @@
+from flask import request, jsonify
+from models import db, Tool, Chemical
```

Ruff (0.11.9)

`models.db` imported but unused

Remove unused import: `models.db`

(F401)